README.md

```diff
@@ -1 +1,56 @@
-# php-scrypt
\ No newline at end of file
+#  scrypt-php
+
+A PHP Extension written in Rust to create scrypt password hashes
+
+## Usage
+Using the scrypt extension is easy, just add the extension to your PHP installation and use the scrypt functions like so:
+```php
+<?php
+/**
+* @param $password The string you want to hash (required)
+* @param $salt The salt you want to use (required)
+* @param $cpu_difficulty The CPU difficulty [default=32768]
+* @param $memory_difficulty The memory difficulty [default=8]
+* @param $parallel_difficulty The parallel difficulty [default=1]
+* @param $len The length of the generated hash [default=8]
+**/
+
+$hash = \scrypt("password", "salt", 32768, 8, 1, 64);
+```
+
+## Building the extension
+Building this extension requires that you have a version of PHP installed that has the `php-config` command.
+
+After all the prequisites are met simply run
```

Comment:
It's suggesting `prequisites` instead of `prequisites`. Dont know the word, please double-check.